### DISCUSSION TITLE
Documentation for Font-Family omits important information

### DISCUSSION THREAD

**Discussion by Johnrobmiller** [tailwindlabs/tailwindcss#3518] [Ideas]
Hello, I am a new user of Tailwind.  The documentation on Font-Family does not mention how to import new font-families.  Thankfully, I managed to find the answer on Stack-Overflow (https://stackoverflow.com/questions/60854215/tailwind-use-font-from-local-files-globally).  However, in my opinion, I don't think it's good policy to rely on random contributions from Stack-Overflow to teach new users how to do common tasks such as importing fonts.

Also, I found something about importing fonts on the base-styles page (https://tailwindcss.com/docs/adding-base-styles#font-face-rules). However, there is a problem with this: new users who are trying to figure out how to import new font-families will not look for the answer in the "base styles" page because new users won't have any idea what "base styles" are.

This might not be a feature-breaking bug, and I'm sure there will be many who dismiss it or don't care, but documentation is extremely important!  After all, a user not knowing how to use a feature (possibly due to problems within the documentation) literally has the same overall result as if the feature were bugged or broken.  Therefore, please consider adding this information to the Font-Family documentation.

**Comment by simonswiss** [comment]
Hey!

Agreed, it's likely confusing that the "importing custom fonts" documentation is not collocated with the [Customizing Font Families](https://tailwindcss.com/docs/font-family#customizing) section of the Tailwind CSS documentation.

It also has its place in the base styles, but we could probably add a paragraph that links to that from the font families page 👍

Thanks for the feedback, it's always useful and welcome!
